api/webhooks/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+
+class AbstractBaseWebhookModel(models.Model):
+    url = models.URLField()
+    secret = models.CharField(max_length=255, blank=True)
```

Comment:
>  Avoid using null on string-based fields such as CharField and TextField. If a string-based field has null=True, that means it has two possible values for “no data”: NULL, and the empty string. In most cases, it’s redundant to have two possible values for “no data;” the Django convention is to use the empty string, not NULL. One exception is when a CharField has both unique=True and blank=True set. In this situation, null=True is required to avoid unique constraint violations when saving multiple objects with blank values.

ref: https://docs.djangoproject.com/en/4.0/ref/models/fields/#null

Reply:
TIL!